### ISSUE TITLE
ERROR: >= 0 (-3 vs. 0)

### ISSUE THREAD

**Issue by aqader** [NVIDIA/DIGITS#1449]
I keep receiving this error when attempting to finetune the GoogLeNet in Caffe. My dataset utilizes 112x112 images. My train prototxt is exactly as the GoogLeNet except for the last softmax layer being retrained. Any ideas?

Thanks.

**Comment by lukeyeager** [comment]
It's possible 112x112 is too small for GoogLeNet. I can't recall exactly. Can you post your `caffe_output.log`?
